charts/workspacemanager/templates/proxy-configmap.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: v1
```

Comment:
General comment about this file: Do you need me to do a close review of this file? If so, it would be helpful if I could view a rendered copy of it.

That said, it looks correct based on a quick pass through it.

Reply:
A quick pass is probably the right level of review for now, especially since I still need to come back and add in tcell configuration at some point.
Thank you for the comments!